ci.template

```diff
@@ -148,21 +148,34 @@
                         }
                     },
                     {
-                        "PolicyName": "publish-metrics",
+                      "PolicyName": "publish-metrics",
+                      "PolicyDocument": {
+                        "Statement": [
+                          {
+                            "Action": [
+                              "s3:PutObject",
+                              "s3:GetObject",
+                              "s3:GetObjectAcl"
+                            ],
+                            "Effect": "Allow",
+                            "Resource": ["arn:aws:s3:::mapbox-loading-dock/raw/mobile.binarysize/*",
+                                       "arn:aws:s3:::mapbox-loading-dock/raw/mobile.codecoverage/*"]
```

Comment:
nit: Is the code coverage resource being used?

Reply:
I don't _think_ so? Looks like I searched in the project locally and couldn't find any instance where that location was being referenced.